backend/db/config.py

```diff
@@ -124,16 +124,14 @@ def get_pg_connect_url(local=False):
     ],
     'codeset_counts': ['members_items_summary'],
     'codeset_ids_by_concept_id': ['cset_members_items'],
-    'concept_ancestor_plus': ['concept_ancestor', 'concepts_with_counts', 'concept_depth'],
-    'concept_depth': ['root_concepts', 'concept_ancestor'],
+    'concept_ancestor_plus': ['concept_ancestor', 'concepts_with_counts'],
```

Comment:
## Delete: `concept_depth` and `root_concepts` from dependency config
These are temp tables.

Deleting these from config not only reduces unnecessary operations during the refresh, but fixes an error.